source/ai-data-usage.txt

```diff
@@ -0,0 +1,36 @@
+.. vscode-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The MongoDB Github Copilot Participant is powered by Generative AI (Gen
+AI), and may give inaccurate responses. See our `Generative AI FAQ
+<https://dochub.mongodb.org/core/faq-ai-features>`__ for more
+information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+The MongoDB Github Copilot Participant is powered by `Github Copilot <https://github.com/features/copilot>`__.
+This is subject to change in the future. 
+
+How Your Data is Used
+---------------------
+
+When you use the MongoDB Github Copilot Participant, the following
+information is sent to MongoDB's backend and/or the third party AI
```

Comment:
The 'third party AI provider' here refer distinct to two things:
1. The LLM API exposed by Copilot itself (we theoretically use the exact same LLM that Copilot uses)
2. Our docs chatbot API endpoint used by our Docs Chatbot

Reply:
Not sure if we need to be that specific, however,

Reply:
@themantissa What do you think?